applications/location-api/README.md

```diff
@@ -0,0 +1,171 @@
+
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
```

Comment:
Travis CI is failing on this copyright header. The correct syntax should be:

~~~
```text
SPDX-License-Identifier: Apache-2.0
Copyright (c) 2020 LINKS Foundation
```
~~~